### PR TITLE
Combining .action-bar, .section.do, .section.start. closes #2211

### LINKED ISSUE #2205: Combine #content .section.start and #content .section.do styles
They're basically the same pattern. Let's consolidate.

### LINKED ISSUE #2211: Combine .action-bar, .section.do, and .section.start?
Related to #2205, #2091, #2188. Blocks #2090. 

The `.action-bar` and `.section.do` bars are more or less the same component with the button floated to the left or right. `.section.apply` is similar, but green. Right now, they're styled slightly differently. I think we can standardize them, and reduce the CSS involved in all three.

The `.action-bar` is the gray bar at the top of [Disability Benefits > Conditions](https://vets.gov/disability-benefits/conditions/)

screen shot 2016-05-02 at 6 21 42 pm

----

**`.section.do`** is the gray bar found at the bottom of [Disability Benefits](https://vets.gov/disability-benefits/) 

screen shot 2016-05-02 at 6 21 37 pm

----

**`.section.start`** is the green bar found at the bottom of the [Disability Benefits > Apply for Benefits](https://www.vets.gov/disability-benefits/apply-for-benefits/) page.

screen shot 2016-05-02 at 6 32 15 pm

### PR DESCRIPTION
**Pull request by webinista**
closes #2205
- Removing .actions child of .section.do component.
- Changing .section.do to .va-action-bar--footer.
- Changing .action-bar to .va-action-bar--header.
- Changing .section.start to .va-action-bar--start.

### CODE REVIEW

**Comment by webinista** [comment]
This is a _really_ large pull request that affects a lot of disability benefits and education pages, but it's the same change in almost every place: changing `action-bar` to `va-action-bar--header` or `class="section do"` to `class="va-action-bar--footer"`.

File: _disability-benefits/conditions.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ relatedlinks:
 
 <div class="main" role="main" markdown="0">
 
-<div class="action-bar">
+<div class="va-va-action-bar--header--header">
```
Comment (emilyville) [inline _disability-benefits/conditions.md]: double header!

File: _sass/_va.scss [hunk 2]
```diff
@@ -2199,14 +2183,31 @@ h4.highlight {
 }
 
 // Action banner
+//.action-bar {
+//  text-align: right;
+//  background: $color-gray-lightest;
+//  padding: .5em 0;
+//}
```
Comment (emilyville) [inline _sass/_va.scss]: did you mean to leave this in, but commented out?

**Comment by emilyville** [comment]
one small thing, and one question!

**Comment by webinista** [comment]
Fixed those two issues. Thanks for catching them!

**Comment by emilyville** [comment]
LGTM!

**Pull request merged**